docs/quickstarts/fre-quickstart.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Quick start with first-run experience
+description: Learn how to implement a first-run experience for your add-in.
+ms.date: 08/08/2024
+ms.topic: how-to
+ms.localizationpriority: medium
+---
+
+# Build an add-in with a basic first-run experience
+
+In this article, you'll walk through the process of building a task pane add-in that implements a [first-run experience](../design/first-run-experience-patterns.md) using the value placemat pattern. When the user runs the add-in, the add-in determines whether or not to show the first-run experience by checking local storage for the existence of a certain flag.
+
+This quick start provides instructions and screenshots for Excel but you can use a similar pattern to implement a first-run experience in other Office applications where Office Web Add-ins are supported. If you'd like to use a different Office application, see that application's quick start for setup instructions before proceeding. Also note that actual interactions with your selected Office application will differ.
+
+## Prerequisites
+
+[!include[Yeoman generator prerequisites](../includes/quickstart-yo-prerequisites.md)]
+
+## Create the add-in project
+
+[!include[Yeoman generator create project guidance](../includes/yo-office-command-guidance.md)]
+
+- **Choose a project type:** `Office Add-in Task Pane project`
+- **Choose a script type:** `JavaScript`
+- **What do you want to name your add-in?** `My Office Add-in`
+- **Which Office client application would you like to support?** `Excel`
+
+![The Yeoman Generator for Office Add-ins command line interface.](../images/yo-office-excel.png)
+
+After you complete the wizard, the generator creates the project and installs supporting Node components.
+
+[!include[Node.js version 20 warning](../includes/node-20-warning-note.md)]
+
+[!include[Yeoman generator next steps](../includes/yo-office-next-steps.md)]
+
+## Explore the project
+
+[!include[Yeoman generator add-in project components](../includes/yo-task-pane-project-components-js.md)]
+
+## Try it out
```

Comment:
Let's remove this first try it out section. There's no point in running the add-in before the FRE is implemented.